File: Seminar/src/main/java/org/sopt/Seminar/domain/post/repository/PostRepository.java
package org.sopt.Seminar.domain.post.repository;

import java.util.List;
import org.sopt.Seminar.domain.member.model.Member;
import org.sopt.Seminar.domain.post.model.Post;
import org.springframework.data.jpa.repository.JpaRepository;

public interface PostRepository extends JpaRepository<Post, Long> {
    List<Post> findAllByMember(Member member);
}

File: kopring/week2/kotlin-semina/src/main/kotlin/org/sopt/kotlinsemina/domain/post/repository/PostRepository.kt
package org.sopt.kotlinsemina.domain.post.repository

import org.sopt.kotlinsemina.domain.member.model.Member
import org.sopt.kotlinsemina.domain.post.model.Post
import org.springframework.data.jpa.repository.JpaRepository

interface PostRepository : JpaRepository<Post, Long> {
    fun findAllByMember(member: Member) : List<Post>
}